ansible/roles/biocache-service/vars/main.yml

```diff
@@ -15,3 +15,4 @@ biocache_service_war_path: "{{  biocache_service_maven_repo_ws_url | default(mav
 additional_proxy_pass:
   - { src: "/biocache-download", dest: "!" }
 
+content_type_options_header: "{{ biocache_service_content_type_options_header | default(false) }}"
```

Comment:
`content_type_options_header` is true in nginx by default. This is false in some services because of this:
https://github.com/AtlasOfLivingAustralia/regions/issues/63